examples/QuickStart/JavaSpring/QuickStart/src/main/java/app/config/QuickStart/MyProperties.java

```diff
@@ -0,0 +1,15 @@
+package app.config.QuickStart;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "config")
```

Comment:
We are thinking we should add some code comments.

```suggestion

// The `prefix` value of `@ConfigurationProperties` matches the created key of `/application/config.message` so all properties with that prefix are added to this `@ConfigurationProperties`
// Note: By default, `/application/` is removed from the key name.
@ConfigurationProperties(prefix = "config")
```